src/utilities/processo-juridico/index.ts

```diff
@@ -21,3 +21,34 @@ export function format(processoJuridico: string) {
       return result;
     }, '');
 }
+
+export function verifyDigit(processo: string): boolean {
+  const digits = processo.split('');
+  const verificationDigits = digits.splice(7, 2).join('');
```

Comment:
Maybe you could put `7` and `2` in variables to give some description of what they are

Reply:
Sure, also moved them into variables!